libraries/standard/http/cbmc/proofs/HTTPClient_AddHeader/Makefile

```diff
@@ -0,0 +1,15 @@
+HARNESS_ENTRY=harness
+HARNESS_FILE=HTTPClient_AddHeader_harness
+
+DEFINES +=
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY +=
+UNWINDSET += strncmp.0:5
+
+PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c 
+PROOF_SOURCES += $(PROOFDIR)/../../sources/http_cbmc_state.c
+
```

Comment:
Might want to delete blank link to be consistent with your other Makefile.

Reply:
Wow how'd you catch that haha